package.json

```diff
@@ -17,10 +17,14 @@
     "debug": "4.0.1",
     "dotenv": "6.0.0",
     "express": "4.16.3",
+    "express-jwt": "^5.3.1",
```

Comment:
@marcgardiner It's not good to use mix of caret(^)+tilde(~) versions and exact versions. Better to use **either** only exact versions **or** caret+tilde versions.

Reply:
@igortalent Fixed with https://github.com/marcgardiner/zi-spa/pull/16/commits/6cb33b6dec89df2b9d16fdc8c4166bcaa54ab702